flycheck.el

```diff
@@ -4489,6 +4529,9 @@ symbols in the command."
     (condition-case err
         (let* ((program (flycheck-checker-executable checker))
                (args (flycheck-checker-substituted-arguments checker))
+               (command (funcall flycheck-command-wrapper-function
```

Comment:
I've thought about this again.  Why don't we pass `(cons program args)` to the wrapper function, whose default value would then be `#'identity`?  That'd make for a much nicer API in my opinion, what do you think?